create-astronomical-data

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
```

Comment:
```suggestion
#!/usr/bin/env python3
```
(you never know 😅 )

Reply:
I believe this file is not tested.

I believe splitting the script into a function that does the computation and a function that does the storage would make things easier to test

Also, I think it would make sense if the function to read from and write to gzipped json files were written next to each other.